app/views/wiki/show.html.erb

```diff
@@ -43,7 +43,7 @@
       <ul class="nav nav-tabs">
         <li class="active"><a href="#"><span class="hidden-xs"><%= t('wiki.show.view') %></span><span class="visible-xs"><i class="fa fa-file-o"></i></span></a></li>
         <% if @node.has_tag('locked') %>
-          <li><a href="/wiki/locked"><i class="fa fa-lock"></i></a></li>
+          <li><a href="/wiki/edit/pagetitle"><i class="fa fa-lock"></i></a></li>
```

Comment:
And we should add a conditional -- `<% if current_user && (current_user.role == "moderator" || current_user.role == "admin") %>` with the "real" link, but if not, go to `/wiki/locked` -- make sense?